app/src/main/java/com/stevesoltys/seedvault/restore/RestoreSetFragment.kt

```diff
@@ -50,6 +51,10 @@ class RestoreSetFragment : Fragment() {
 
         skipView.setOnClickListener {
             viewModel.onFinishClickedAfterRestoringAppData()
+            // Equivalent to com.google.android.setupcompat.util.ResultCodes.RESULT_SKIP
+            // SetupWizard handles this
+            requireActivity().setResult(RESULT_FIRST_USER)
+            requireActivity().finishAfterTransition()
```

Comment:
If we finish here, how will the user be able to restore files?

Reply:
Good point. 

https://github.com/seedvault-app/seedvault/blob/129f8d90e5862a3614f11e187d85ca1022e08701/app/src/main/java/com/stevesoltys/seedvault/restore/RestoreFilesFragment.kt#L35-L38

Could change the `RESULT_OK` there.

And I just noticed that uses `.apply()`, which is idiomatic kotlin style yes?